client/src/App.js

```diff
@@ -40,7 +40,7 @@ function App() {
     axios({
       method: "GET",
       withCredentials: true,
-      url: process.env.PROD ? "http://halogen-web-app.herokuapp.com:80/user" : "http://localhost:3001/user",
+      url: "/user",
```

Comment:
Just wanted to point this out @Hanna-N9 -

We were trying to reference the `process.env` variable on the client side, and it is only available on the server. For these URL's, we don't need to specify the absolute path, it can be relative like `/user`.